### ISSUE TITLE
Add event date in other places

### ISSUE THREAD

**Issue by fzipi** [OpenSecuritySummit/oss2018#262]
There were some people confused by the date of the event, because it only show on the first slide of the carousel... 

We should add a clear date reference in  other places. Maybe a footer, on the side or other places.
Suggestions?

**Comment by VasBu** [comment]
What about adding it in the header? Whatever page will be opened the visitor will be able to see from 04 to 08 June?

![image](https://user-images.githubusercontent.com/15072044/39762967-3c6fa072-52dc-11e8-8b64-0788bb5eadd6.png)

**Comment by fzipi** [comment]
@Lisa-devseccon Will the solution of adding the date to the header work for you? Wouldn't be too crowded?

**Comment by Lisa-devseccon** [comment]
@fzipi I agree, date should be in header. can you please integrate it like this:
![date-in-header](https://user-images.githubusercontent.com/27144792/40288679-a6942c70-5cb4-11e8-9db6-3d17cd11bc50.png)
